docs/reference/sql/appendix.md

```diff
@@ -19,7 +19,7 @@ The following table shows all keywords in the language.
 | `AND`          | logical "and" operator                  | `WHERE userid<>'User_1' AND userid<>'User_2'`                        |
 | `ARRAY`        | one-indexed array of elements           | `SELECT ARRAY[1, 2] FROM s1 EMIT CHANGES;`                           |
 | `AS`           | alias a column, expression, or type     |                                                                      |
-| `ASSERT`       | assert value, stream, table, tombstone  | `ASSERT NULL VALUES sourceName (columns)? KEY values;`               |
+| `ASSERT`       | assert value, stream, table, tombstone, topic, schema  | `ASSERT NULL VALUES sourceName (columns)? KEY values;`|
```

Comment:
Should the optional timeouts be mentioned?  I suppose I'm unclear on the point of this page.  Is it examples or more like an EBNF?

Reply:
Yeah, thanks for catching that. It's a list of all keywords in ksqlDB's grammar.

Reply:
Gotta earn that reviewer cred somehow:).